apps/docs/src/getting-started.md

```diff
@@ -4,8 +4,12 @@
 This library is currently in early / active development.
 :::
 
+<div class="external-link--hide">
```

Comment:
strict BEM would need
```suggestion
<div class="external-link external-link--hide">
```

Reply:
I renamed it to `hide-external-link` so no BEM is needed :D

Reply:
ok :)